docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -5,48 +5,165 @@
 
 ### Objective
 
-In order for a domain to work, it must be delegated to from a domain higher
-up in the DNS hierarchy (a parent domain). However, note that this
-requirement does not concern the root domain.
+In order for a domain (zone) to work, it must be delegated to from a 
+domain (zone) higher up in the DNS hierarchy (a parent domain/zone). 
+This Test Case will determine if the parent zone exists and if the
+child zone exists (is delegated).
+
+If the zone to be tested is the root zone it has no parent and no
+delegation and will always pass this Test Case.
+
+If the child zone does not exists, then only BASIC03 will be run,
+unless the test type is an undelegated test, which can always be 
+run.
 
 ### Inputs
 
-The label of the domain name to be tested.
+Input for this Test Case:
+* The label of the domain name (zone) to be tested ("child zone").
+* The fact if the test type is undelegated test ("undelegate test") or not
+  ("normal test").
```

Comment:
The fact if the test type is undelegated test ("undelegate test") or not  ("normal test"). ==> If the test type is undelegated, then "undelegate test"

Reply:
I agree that "normal test" is the default one (and it is called "normal"), but in the specification we must be able to refer to both types of tests. That is why I want to introduce it in the input.